plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHivePipelinedExecutionConnectorTest.java

```diff
@@ -14,12 +14,15 @@
 package io.trino.plugin.hive;
 
 import com.google.common.collect.ImmutableMap;
+import io.trino.testing.QueryRunner;
 
 public class TestHivePipelinedExecutionConnectorTest
```

Comment:
what does "pipelined execution" mean here?

it this is "test hive with defaults", let's call the class just `TestHiveConnectorTest`

Reply:
Yeah - effectively it is with defaults (no failure recovery). I will rename in a separate PR after this gets merged.